guides/v2.2/advanced-reporting/report-xml.md

```diff
@@ -25,7 +25,7 @@ Columns are added using the `<attribute>` node.
 
 All report files are located in the `etc` directory of a module:
 
-```
+```terminal
```

Comment:
This is not supposed to be a terminal output. For such cases, it makes sense to use the `text` [lexer](https://github.com/rouge-ruby/rouge/wiki/List-of-supported-languages-and-lexers).

Reply:
Fixed